go.mod

```diff
@@ -42,15 +42,15 @@ require (
 	github.com/spf13/afero v1.2.2
 	github.com/spf13/cobra v1.1.3
 	github.com/yuin/goldmark v1.3.3
-	go.opencensus.io v0.22.4
+	go.opencensus.io v0.23.0
 	go.uber.org/zap v1.10.0
 	goji.io v2.0.2+incompatible
 	golang.org/x/crypto v0.0.0-20210513164829-c07d793c2f9a // indirect
-	golang.org/x/mod v0.3.0
-	golang.org/x/oauth2 v0.0.0-20210113205817-d3ed898aa8a3
-	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
-	google.golang.org/api v0.30.0
-	google.golang.org/grpc v1.31.0
+	golang.org/x/mod v0.4.1
+	golang.org/x/oauth2 v0.0.0-20210313182246-cd4f82c27b84
+	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
+	google.golang.org/api v0.42.0
```

Comment:
this is the update that we needed to fix in build bot for https://github.com/solo-io/envoy-gloo/pull/131/files